crypto/x509/v3_admis.c

```diff
@@ -81,14 +80,14 @@ static int i2r_NAMING_AUTHORITY(const struct v3_ext_method *method, void *in,
         && namingAuthority->namingAuthorityUrl == NULL)
         return 0;
 
-    if (BIO_printf(bp, "%*snamingAuthority: ", ind, "") <= 0)
+    if (BIO_printf(bp, "%*s  namingAuthority:\n", ind, "") <= 0)
```

Comment:
If you feel the need to add spaces on the first line printed in any `i2r` function, then it's more likely that it's being called with too little indentation.  And indeed, lines 146 and 164 are calling `i2r_NAMING_AUTHORITY()` with too little indentation.

So please undo the added spaces in this function and correct the calls instead

Reply:
@levitte ah I see, thank you for the hint! I have reverted the previous commit and pushed a new one with increasing the space count instead.